android/src/test/java/com/instabug/reactlibrary/RNInstabugReactnativeModuleTest.java

```diff
@@ -205,11 +205,26 @@ public void tearDown() {
 
         String email = "sali@instabug.com";
         String userName = "salmaali";
+        String id = null;
         // when
-        rnModule.identifyUser(email, userName);
+        rnModule.identifyUser(email, userName, id);
         // then
         verify(Instabug.class,times(1));
-        Instabug.identifyUser(userName, email);
+        Instabug.identifyUser(userName, email, id);
+    }
+
+    @Test
+    public void givenArgs$identifyUserwithId_whenQuery_thenShouldCallNativeApiWithArgs() {
+        // given
+
+        String email = "sali@instabug.com";
+        String userName = "salmaali";
+        String id = "salmaali";
+        // when
+        rnModule.identifyUser(email, userName, id);
+        // then
+        verify(Instabug.class,times(1));
+        Instabug.identifyUser(userName, email, id);
```

Comment:
I think let's refactor this test to be cleaner like we do with Flutter, the old format feels weird TBH.

Let's also rename the test to something better than `givenArgs$identifyUserwithId_whenQuery_thenShouldCallNativeApiWithArgs`